### ISSUE TITLE
UserPasswordEncoderListener.groovy: 8: unable to resolve class grails.events.annotation.gorm.Listener

### ISSUE THREAD

**Issue by muehlboeck** [grails-plugins/grails-spring-security-core#996]
### Expected Behavior

application should run with 
./gradlew bootRun

### Actual Behaviour

./gradlew bootRun fails with:
UserPasswordEncoderListener.groovy: 8: unable to resolve class grails.events.annotation.gorm.Listener

### Steps To Reproduce

sdk use grails 6.1.1 
sdk use groovy 3.0.11 
sdk use java 11.0.21-sem

grails create-app test
cd test
create file grails-app/conf/spring/resources.groovy
with content:
beans = {
}

./gradlew runCommand "-Pargs=s2-quickstart me.test.main User Role"

will complete 

but:

./gradlew bootRun

fails 

### Environment Information

linux 22.04
sdk use grails 6.1.1 
sdk use groovy 3.0.11 
sdk use java 11.0.21-sem

### Version

6.1.1

**Comment by matrei** [comment]
Should be fixed when 6.1.2 is released.

Can be worked around by adding:
```groovy
implementation 'org.grails.plugins:events:5.0.2'
implementation 'org.grails.plugins:async:5.0.2'
```

Related grails/grails-core#13324 and grails/grails-core#13331

**Comment by muehlboeck** [comment]
Fantastic. This works for now. Thanks.

**Comment by ewcole** [comment]
It is still a problem in Grails 6.2.2.  Fortunately, matrei's workaround is still a solution.

**Comment by matrei** [comment]
The actual problem is in `grails-spring-security-core`. It is fixed in #1005 but there has not been a release with that change. If you update to `org.grails.plugins:spring-security-core:6.1.2-SNAPSHOT` it should work.

**Comment by jamesfredley** [comment]
org.grails.plugins:spring-security-core:6.1.2 has been released
